js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -715,7 +715,7 @@ addCustomProvider=Add custom provider
 permissionType=Specifies that this permission must be applied to all resources instances of a given type.
 policyEnforcementModes.ENFORCING=Enforcing
 rowSaveBtnAriaLabel=Save edits for {{messageBundle}}
-permanentLockout=Permanent lockout
+maxTemporaryLockouts=Maximum temporary lockouts
```

Comment:
Perhaps we should explain that `0` will permanently lock the user and `-1` will disable it? That is what I meant by not sure about the best UX/UI.

Reply:
I was planning on putting that explanation into a tooltip. Thanks for the reminder.